eventkit_cloud/jobs/admin.py

```diff
@@ -129,8 +129,8 @@ class Meta:
 
     def clean_config(self):
         config = self.cleaned_data.get('config')
-        if config == "":
-            return self.cleaned_data
+        if not config:
+            return
```

Comment:
Since it's a cleaning function that's meant to return a result, `return config` might be more appropriate than `return`. But if it's not causing exceptions, then the calling function is able to handle `None` instead of `""`, so it probably makes little difference.